packages/buidler-core/src/internal/buidler-evm/provider/node.ts

```diff
@@ -985,6 +978,30 @@ export class BuidlerNode extends EventEmitter {
     return logs;
   }
 
+  public async addCompilationResult(
+    compilerVersion: string,
+    compilerInput: CompilerInput,
+    compilerOutput: CompilerOutput
+  ): Promise<boolean> {
+    let bytecodes;
+    try {
+      bytecodes = createModelsAndDecodeBytecodes(
+        compilerVersion,
+        compilerInput,
+        compilerOutput
+      );
+    } catch (error) {
+      log("Compilation result could not be added.\n", error);
```

Comment:
Is this log fine?

Reply:
Shouldn't we do something similar to what we do when initializing Buidler EVM for the first time? We also print a warning there.

Reply:
I added similar messages to those, replacing "initialized" with "updated" and removing the part about the tracing being disabled since it doesn't make sense here.